lib/logflare/backends/adaptor/tcp_adaptor.ex

```diff
@@ -0,0 +1,126 @@
+defmodule Logflare.Backends.Adaptor.TCPAdaptor do
```

Comment:
would be more accurate to call this SyslogAdaptor, since TCPAdaptor would be more generic and wouldnt care about  the message format.
we can refactor this into something like HttpBased in the future